pkg/asset/tls.go

```diff
@@ -98,18 +109,55 @@ func newAPIKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey, altNa
 	return key, cert, err
 }
 
-func newKubeletKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey) (*rsa.PrivateKey, *x509.Certificate, error) {
+func newAdminKeyAndCert(caCert *x509.Certificate, caPrivKey *rsa.PrivateKey) (*rsa.PrivateKey, *x509.Certificate, error) {
 	key, err := tlsutil.NewPrivateKey()
 	if err != nil {
 		return nil, nil, err
 	}
 	config := tlsutil.CertConfig{
-		CommonName:   "kubelet",
-		Organization: []string{"kube-node"},
+		CommonName:   "cluster-admin",
+		Organization: []string{"system:masters"},
 	}
 	cert, err := tlsutil.NewSignedCertificate(config, key, caCert, caPrivKey)
 	if err != nil {
 		return nil, nil, err
 	}
 	return key, cert, err
 }
+
+// newBootstrapAuthTokenFile creates a static token file, containing a single
+// randomly-generated bearer token mapped to the user and group
+// 'kubelet-bootstrap'.
+//
+// This token file is meant to be used by kubelet to bootstrap its TLS
+// certificate as described on k8s.io/docs/admin/kubelet-tls-bootstrapping.
+func newBootstrapAuthTokenFile() ([]byte, error) {
```

Comment:
I consciously named this function with the `File` prefix to make a distinction between a simple token / bearer (e.g. _487afc487_ ) and something that follows the token file format defined by Kubernetes' `plugin/pkg/auth/authenticator/token/tokenfile/...`. Renaming and making clear that we follow that format in the comment.